modules/Bio/Vega/Utils/Comparator.pm

```diff
@@ -53,8 +53,8 @@ sub compare {
         my $class = ref($obj1);
         throw("I need to run 'vega_hashkey' method on the objects, which is not available for class $class.");
     }
-    my $obj1_hash_key=$obj1->vega_hashkey;
-    my $obj2_hash_key=$obj2->vega_hashkey;
+    my $obj1_hash_key= lc $obj1->vega_hashkey;
+    my $obj2_hash_key= lc $obj2->vega_hashkey;
```

Comment:
Unless some tests of vega_hashkey are case sensitive, it might be better to have `lc` within the hashkey function